front/temporal/labs/activities.ts

```diff
@@ -278,21 +318,15 @@ export async function processGoogleDriveTranscriptActivity(
       return innerArray.find((item) => item.type === "agent_message");
     }
   );
-  const fullAnswer = agentMessage ? agentMessage[0].content : "";
-
-  const hasExistingHistory =
-    await transcriptsConfiguration.fetchHistoryForFileId(fileId);
-  if (hasExistingHistory) {
-    localLogger.info(
-      "[processGoogleDriveTranscriptActivity] History record already exists. Stopping."
-    );
-    return;
-  }
+  const markDownAnswer =
+    agentMessage && agentMessage[0].content ? agentMessage[0].content : "";
+  const htmlAnswer = marked.parse(markDownAnswer);
```

Comment:
⚠️ This does not sanitize the HTML, let's use another library to take care of this.

Reply:
Ok, was wondering if we needed sanitation as we're sending emails and not displaying in browser directly. I searched for a sanitation lib we're already using, seems like we're not using any. 
Will follow marked's reco using `DOMPurify` if that's OK with you.